### PR TITLE
User story 17/view past prompts

### LINKED ISSUE #17: As a player, I want to see my past prompt so that I remember a past prompt.

### PR DESCRIPTION
**Pull request by RudolphTorlage**
The ability for users to view prompts from previous games has been added. Created a History button for users to access functionality. Guests will simply see an empty list. 
This also connects the application to an online database.
Basic Session Management has been implemented.

closes #17

### CODE REVIEW

File: src/controller/dbController.js
Comment (ChristopherPearse) [inline src/controller/dbController.js:1]: Are commented out sections to be added again later? If not please consider reducing clutter in the workspace.

File: tests/test-1.spec.ts
Comment (ChristopherPearse) [inline tests/test-1.spec.ts:1]: Satisfactory use of test cases, however please use the naming convention that was used throughout the rest of the project for test files.

Comment (ChristopherPearse) [reply tests/test-1.spec.ts:1]: Subsequently the tests failed on various web browsers. Please amend before requesting to merge again.

**Review by ChristopherPearse — CHANGES_REQUESTED** [review]
Please address the concerns that I have illustrated through the comments. Overall the work was good however a  few formatting errors, as well as unit test errors.

File: src/controller/dbController.js
Comment (RudolphTorlage) [reply src/controller/dbController.js:1]: they will be added back in after all the other tests have been adjusted to go through the process of logging in to view certain pages

**Comment by RudolphTorlage** [comment]
tests were failing due to secrets not being correctly added to github. This has been resolved.

**Review by ChristopherPearse — APPROVED** [review]
All issues from previous review have been resolved.

Code is of a good standard.

Test cases all pass.

**Pull request merged**